### PR TITLE
Update to work with MBlast 2.0

### PR DESCRIPTION
**Pull request by Foxcapades**
Updates the MultiBlast client to work with the 2.0 rewrite of the MB service.

Changes:

* Split the API wrapper client into 2 separate clients to match the splitting of the MB service into 2 separate services.
* Grouped param constants into a const object named `ParamNames`.

### CODE REVIEW

File: src/lib/components/BlastWorkspaceResult.tsx [hunk 1]
```diff
@@ -389,18 +399,22 @@ function BlastSummary({
...
-              ? jobDetails.config.tool
-              : jobDetails.config.task}
+            {jobDetails.blastConfig.tool === 'deltablast' ||
+            jobDetails.blastConfig.tool === 'psiblast' ||
+            jobDetails.blastConfig.tool === 'rpsblast' ||
+            jobDetails.blastConfig.tool === 'rpstblastn' ||
+            jobDetails.blastConfig.tool === 'tblastx' ||
+            jobDetails.blastConfig.task == null
+              ? jobDetails.blastConfig.tool
+              : jobDetails.blastConfig.task}
```
Comment (dmfalke) [inline src/lib/components/BlastWorkspaceResult.tsx:417]: This could use a comment. Why do we need this logic?

**Review by dmfalke — COMMENTED** [review]
This looks great. Very nicely organized!

I have a few general comments/suggestions, in addition to an inline comment:
1. Kudos on the `ParamNames` object. 
2. Rename `utils/api/{query,blast}/api` to `utils/api/{query,blast}/types`
3. Have verbiage changes (e.g., "description" => "summary") been discussed elsewhere?
4. `taggedFetch` is clever. This seems like something we could incorporate into `FetchClient`.
5. The `io-ts` docs discourage unions of string literals ([source](https://github.com/gcanti/io-ts/blob/master/index.md#union-of-string-literals)).

**Comment by Foxcapades** [comment]
> 3\. Have verbiage changes (e.g., "description" => "summary") been discussed elsewhere?

It was not discussed beyond the approvals on the API design.  The change is not reflected in the UI.

**Comment by Foxcapades** [comment]
> 4\. `taggedFetch` is clever. This seems like something we could incorporate into `FetchClient`.

`taggedFetch` is copied over from the original API client, so credit for that goes to Jamie.

**Comment by Foxcapades** [comment]
> 2\. Rename `utils/api/{query,blast}/api` to `utils/api/{query,blast}/types`

Done.

File: src/lib/components/BlastWorkspaceResult.tsx (hunk 1 above)
Comment (Foxcapades) [reply src/lib/components/BlastWorkspaceResult.tsx:417]: The property `task` is undefined on most blast config types.  I guess this is showing the `task` property value if available, and if not, falling back to the `tool` property value.  I'll comment on it.

**Review by dmfalke — APPROVED** [review]
Thanks for making these changes. You are good to proceed with merging this PR. Then next step would be to increment the package version, ideally using this commmand:
```
yarn version --major
```

That will increment the major version of the package, commit the change, and add a git tag to the commit. Once you push that change, you can go ahead and merge.

**Pull request merged**